cypress/integration/tip.test.ts

```diff
@@ -89,17 +107,13 @@ describe('Swap::Trade::Tip', () => {
           expect(amountNoFee.toSignificant(inputAmount.currency.decimals)).to.not.be.null
           expect(amountWithFee.toSignificant(inputAmount.currency.decimals)).to.not.be.null
 
-          // Trade should have input/out amounts
-          expect(trade).to.have.property('inputAmount')
-          expect(trade).to.have.property('outputAmount')
-
           // input/output amounts should not be equal (ETH/DAI)
           expect(inputAmount.toExact()).to.not.equal(outputAmount.toExact())
 
-          // Subtract amountNoFee from amountWithFee to get the feeAmount
-          expect(amountWithFee.subtract(amountNoFee).toSignificant(inputAmount.currency.decimals)).to.equal(
-            feeAsCurrency?.toSignificant(inputAmount.currency.decimals) || '0'
-          )
+          const expectedFee = amountWithFee.subtract(amountNoFee).toSignificant(inputAmount.currency.decimals)
+          const calculatedFee = feeAsCurrency?.toSignificant(inputAmount.currency.decimals) || '0'
```

Comment:
I don't believe this optional value should be here.
The test should either expect a null value or not.
I would prefer a test where `feeAsCurrency` is null and another when it's not, to verify both paths.